### ISSUE TITLE
Can't work on real device

### ISSUE THREAD

**Issue by jo3ght** [Alezhka/flutter_incoming_call#2]
Hi ! 
I really happy with this package but it's just working when i debug in simulator and does not working when i debug on real device.

**Comment by CuongNV12** [comment]
I also had the same problem with my Galaxy S10+

**Comment by leech001** [comment]
And I had the same problem with my Xiaomi Mi Note 3.

**Comment by Alezhka** [comment]
Hi, thx for report. I saw on more devices in next few days.

**Comment by hanyska** [comment]
+1

**Comment by Alezhka** [comment]
Hi everyone. Please check 0.0.2+1 version and give feedback worked or not.

**Comment by CuongNV12** [comment]
it works on Galaxy S6 (Android 7), but not on Galaxy S10+ (Android 11). 
-----------------------------------------------------------------------------------------
android.content.ActivityNotFoundException: No Activity found to handle Intent { act=com.github.alezhka.flutter_incoming_call.activity.ACTION_INCOMING_CALL
-----------------------------------------------------------------------------------------
Is it missing something? How do i get the below in my project? 

<activity
            android:name="com.github.alezhka.flutter_incoming_call.IncomingCallActivity"
            android:theme="@style/AppCompatTheme"
            android:screenOrientation="portrait">
            <intent-filter>
                <action android:name="com.github.alezhka.flutter_incoming_call.activity.ACTION_INCOMING_CALL" />
                <category android:name="android.intent.category.DEFAULT" />
            </intent-filter>
        </activity>

        <receiver android:name="com.github.alezhka.flutter_incoming_call.CallBroadcastReceiver"
            android:enabled="true"
            android:exported="false"/>
-----------------------------------------------------------------------------------------

**Comment by Alezhka** [comment]
@CuongNV12 You need to add to your AndroidManifest.xml (See example/android/app/src/main/AndroidManifest.xml). Today I will add this step to the Readme.

**Comment by isuru19963** [comment]
When the app is closed not work incoming call UI. Emulator works fine but in real device not working.

**Comment by Alezhka** [comment]
@leech001 Hi! Please tell me on which platform and os version you find bug. Also indicate the steps to reproduce.

**Comment by leech001** [comment]
> @leech001 Hi! Please tell me on which platform and os version you find bug. Also indicate the steps to reproduce.

Mi Note 3
Android 9

**Comment by pacifio** [comment]
Yep this does not work on real devices , tried it on Samsung M31 (Android 11)

**Comment by pacifio** [comment]
I think an update for Android 11 is needed , not sure how

**Comment by Alezhka** [comment]
@pacifio Please write steps to reproduce bug.

**Comment by vietkeep** [comment]
Work with my android, xiaomi note 9 pro
not working in iphone 7

**Comment by Alezhka** [comment]
@vietkeep Hi. Need more information. Please write steps to reproduce bug. Have you tried running the example project?
